packages/devtools_extensions/extension_config_spec.md

```diff
@@ -3,7 +3,7 @@ The `config.yaml` file for a DevTools extension must follow the format below.
 ## Required fields
 
 - `name` : the package name that this DevTools extension belongs to. The value of this field
-will be used in the extension page title bar.
+will be used in the extension page title bar. This name should not contain spaces or special characters.
```

Comment:
Does "special characters" include underscores? If so, if your package is called "foo_bar", what should you put here (it says "package name").